avm/res/search/search-service/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 The latest version of the changelog can be found [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/search/search-service/CHANGELOG.md).
 
+## 0.11.0
+
+### Changes
```

Comment:
Should we list api version changes as well?

Reply:
If it were the main resource, probably yes. But in this case it's for an `existing` key vault reference. It may make sense that we define some guidlines somewhere.